README.md

```diff
@@ -19,60 +19,120 @@ GeoMesa is an open-source, distributed, spatio-temporal database built on top of
 
 ## Download and Version Information
 
-<b>NOTE:</b> The current recommended version is ```1.1.0-rc1```, which includes breaking changes from ```1.0.0-rc7``` and previous versions due to schema incompatibility and changes in indexing structure. The most recent tar.gz assembly can be 
-[downloaded here](http://repo.locationtech.org/content/repositories/geomesa-releases/org/locationtech/geomesa/geomesa-assemble/1.1.0-rc.1/geomesa-assemble-1.1.0-rc.1-bin.tar.gz) which contains the [Accumulo distributed runtime jar](geomesa-distributed-runtime), [GeoServer plugin](geomesa-plugin), and [command line tools](geomesa-tools).
+<b>NOTE:</b> The current recommended version is ```1.1.0-rc.6```, which includes breaking changes from ```1.0.0-rc.7``` and previous versions due to schema incompatibility and changes in indexing structure. The most recent tar.gz assembly can be 
```

Comment:
I looked into that. The best way we could do it is to have a Maven-filtered README and have it be processed during build time. Otherwise, variables in GitHub markdown are unsupported, unfortunately.